README.md

```diff
@@ -50,15 +50,15 @@ Works with cactus, sugar cane, saplings, hoppers, dispensers and droppers.
 * Categories: `EXTRAS`, `EXPERIMENTAL`  
   
 ## chickenShearing
-Chickens can be sheared to get feathers. Beware! every time u shear a chicken, it gets damaged!  
+Chickens can be sheared to get feathers. Beware! every time you shear a chicken, it gets damaged!  
 Baby chickens can't be sheared.  
 * Type: `boolean`  
 * Default value: `false`  
 * Required options: `true`, `false`  
 * Categories: `EXTRAS`, `FEATURE`  
   
 ## clericsFarmWarts
-Clerics can warm nether farts.  
+Clerics can warm nether warts.  
```

Comment:
This is intentional

Reply:
but why? No offense, but that doesn’t seem like the best idea

Reply:
Well, **warming farts** is faster than **farming warts**. @gnembon has marked multiple similar pull requests and issue reports as WAI (works as intended).

Reply:
um... you’re not wrong, but I think we should go for the more understood option. People may be confused and also, while I’m a fan of humor, we should behave like professionals here.

Reply:
> we should behave like professionals here.

This is literally a block game...

Reply:
Feel free to mark this thread as resolved, it is up to gnembon in the end.